src/style.css

```
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-bar1 {
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: inherit;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar1:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar1:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.search-bar2 {
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  font-style: normal;
+  font-size: 14px;
+  font-weight: 300;
+  font-family: inherit;
+  line-height: 19px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar2:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar2:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.form {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.image-small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search-bar1::placeholder {
+  color: hsl(212, 23%, 31%);
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+}
+
+.search-bar2::placeholder {
+  color: hsl(212, 23%, 31%);
+  font-family: "Avenir", sans-serif;
```

You need to add font-family for base-class, not for placeholders
```suggestion
  font-size: 16px;
  font-style: normal;
}

.search-bar2::placeholder {
  color: hsl(212, 23%, 31%);
```